app/uk/gov/hmrc/eventhub/controllers/PublishController.scala

```diff
@@ -37,8 +38,13 @@ class PublishController @Inject() (cc: ControllerComponents, eventPublisherServi
         errors => Future.successful(BadRequest(Json.obj("Invalid Event payload: " -> JsError.toJson(errors)))),
         event =>
           eventPublisherService.publish(event, topic).map {
-            case Right(subscribers) => Created(Json.toJson(PublishResponse(subscribers.map(_.name))))
+            case Right(subscribers) =>
+              logger.warn(
```

Comment:
you can change to debug and update the app-config-qa application log level to DEBUG temporarily if you want.